packages/arb-token-bridge-ui/scripts/validateOrbitChains.ts

```diff
@@ -0,0 +1,85 @@
+import z from 'zod'
+import { isAddress } from 'ethers/lib/utils.js'
+
+import { getOrbitChains, OrbitChainConfig } from '../src/util/orbit'
+
+const zAddress = z
+  .string()
+  .refine(address => isAddress(address), 'Invalid address')
+
+const zIsTrue = z
+  .boolean()
+  .refine(bool => bool === true, 'Invalid input, must be true')
+
+function validateOrbitChain(chain: OrbitChainConfig) {
+  try {
```

Comment:
now that we're validating this in prebuild, we want it to throw when there's an error, so let's remove try/catch

Reply:
we throw in the catch block, we just want to show info about the chain name/ID first